src/Surfnet/StepupMiddleware/ApiBundle/Identity/Service/RaService.php

```diff
@@ -66,10 +67,10 @@ public function listRas(Institution $institution)
 
         $nameIds = array_merge(
             array_map(function (Ra $ra) {
-                return $ra->nameId;
+                return (string) $ra->nameId;
             }, $ras),
             array_map(function (Raa $raa) {
-                return $raa->nameId;
+                return (string) $raa->nameId;
```

Comment:
`NameId`s were passed to the Identity entity repository, where it actually queries against NameId **strings**.